apps/nextjs-website/src/app/[productSlug]/tutorials/[...productTutorialPage]/page.tsx

```diff
@@ -78,11 +86,61 @@ const Page = async ({ params }: { params: Params }) => {
       bannerLinks={props.bannerLinks}
       showBreadcrumbs={false}
     >
-      <EContainer>
-        <Box sx={{ padding: '56px 0' }}>
-          <GitBookContent content={props.body} config={props.bodyConfig} />
+      <FragmentProvider>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', lg: 'row' },
+            maxWidth: '1156px',
+            margin: '0 auto',
+          }}
+        >
+          <Box
+            sx={{
+              // 78px is the height of the header, 80px is the height of the product header, 24 is the title padding
+              paddingTop: '182px',
+              flexGrow: { lg: 1 },
+              maxWidth: {
+                xs: '100%',
+                lg: '822px',
+              },
+            }}
+          >
+            <GitBookContent content={props.body} config={props.bodyConfig} />
+          </Box>
+          <Box
+            sx={{
+              display: { xs: 'none', lg: 'initial' },
+              position: 'relative',
+              // 78px is the height of the header, 80px is the height of the product header
+              paddingTop: '158px',
+              paddingLeft: '64px',
+              width: { lg: '270px' },
+            }}
+          >
+            <Box
+              sx={{
+                position: 'sticky',
+                maxWidth: '270px',
+                top: 144,
+              }}
+            >
+              <GuideInPageMenu
+                assetsPrefix={props.bodyConfig.assetsPrefix}
+                pagePath={props.path}
+                inPageMenu={props.body}
+                title={translations.productGuidePage.onThisPage}
+              />
+            </Box>
+          </Box>
```

Comment:
The margin at the bottom of the text, when there are no related links is missing:
![image](https://github.com/pagopa/developer-portal/assets/39835990/f9d13689-47d3-4b15-828d-22bb21a40f54)

Look at how it was before:
![image](https://github.com/pagopa/developer-portal/assets/39835990/2c843482-4214-49cb-8441-b473690ae920)

I think you should add a space at the bottom if there are no links